legal-api/report-templates/template-parts/common/businessDetails.html

```diff
@@ -50,6 +50,19 @@
                    {% endif %}
                  </div>
               </td>
+         {% elif header.reportType == 'amalgamationApplication' %}
+            <td>
+               <div class="bold">Incorporation Number:</div>
+               <div class="bold pt-2">Issued Date and Time:</div>
+               <div class="bold pt-2">Recognition Date and Time:</div>
+               <div class="bold pt-2">Retrieved Date and Time:</div>
+            </td>
+            <td>
+               <div>{{business.identifier}}</div>
+               <div class="pt-2">{{effective_date_time}}</div>
+               <div class="pt-2">{{recognition_date_time}}</div>
+               <div class="pt-2">{{report_date_time}}</div>
```

Comment:
recognition_date_time return nothing for me here, so I repeated effective_date_time.